Models/TaskModel.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Models;
+
+class TaskModel
+{
+    public function __construct()
+    {
+        $this->pdo = new \PDO('mysql:host=test.local;dbname=Family', 'snuff', 'kyzmi4');
+    }
+
+    public function createTask($timeCreated, $createdBy, $executor, $task, $status, $timeStart, $timeEnd, $approvedBy, $imgName)
+    {
+        $sql = "INSERT INTO Tasks (time_created, created_by, executor, task, status, time_start, time_end, approved_by, img_name) 
```

Comment:
Make separate class with function insert (like in the project i sent you)
this function must receive something like $tableName, $fields, $values and process it like this.
you can call this class here and call the insert method of this class if u want to add new entries to DB